api/src/schema/index.js

```diff
@@ -64,18 +64,27 @@ const Schema = i18n => {
       results: [Dwelling]
     }
 
+    # ${i18n.t`Walls separate the interior heated space from the outside (interior partition walls are not considered walls)`}
     type Wall @cacheControl(maxAge: 90) {
+      # ${i18n.t`Wall measurements calculated for this evaluation`}
       measurement: WallMeasurement
+      # ${i18n.t`Proposed upgrade wall measurements calculated for this evaluation`}
       upgrade: WallMeasurement
     }
 
+    # ${i18n.t`Wall measurements calculated for this evaluation`}
     type WallMeasurement @cacheControl(maxAge: 90) {
+      # ${i18n.t`Description of wall insulation`}
       insulation: [Insulation]
+      # ${i18n.t`Heat loss through walls in MJ`}
       heatLost: Float
     }
 
+    # ${i18n.t`Material used to insulate something`}
     type Insulation @cacheControl(maxAge: 90) {
+      # ${i18n.t`The proportion of total element that insulation makes up`}
```

Comment:
"proportion of the total element" sounds odd, maybe it's just the "element" part.

Maybe "proportion of the total wall insulation that this element represents", or something similar"

Reply:
It's a bit weird, but Insulation is a general type now so the description has to fit any element it might be used with